### ISSUE TITLE
psycopg2. Programming error -- datcollate

### ISSUE THREAD

**Issue by amelio-vazquez-reina** [dbcli/pgcli#167]
I am running into errors with the latest version of `pgcli`.

When I do:
```
> pgcli -hsome_host.net some_database -Uavazquez -p 54321
```

I get:
```
Traceback (most recent call last):
  File "/Users/avazquez/anaconda3/envs/py34/bin/pgcli", line 9, in <module>
    load_entry_point('pgcli==0.16.0', 'console_scripts', 'pgcli')()
  File "/Users/avazquez/anaconda3/envs/py34/lib/python3.4/site-packages/click/core.py", line 610, in __call__
    return self.main(*args, **kwargs)
  File "/Users/avazquez/anaconda3/envs/py34/lib/python3.4/site-packages/click/core.py", line 590, in main
    rv = self.invoke(ctx)
  File "/Users/avazquez/anaconda3/envs/py34/lib/python3.4/site-packages/click/core.py", line 782, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/avazquez/anaconda3/envs/py34/lib/python3.4/site-packages/click/core.py", line 416, in invoke
    return callback(*args, **kwargs)
  File "/Users/avazquez/anaconda3/envs/py34/lib/python3.4/site-packages/pgcli/main.py", line 344, in cli
    pgcli.run_cli()
  File "/Users/avazquez/anaconda3/envs/py34/lib/python3.4/site-packages/pgcli/main.py", line 170, in run_cli
    self.refresh_completions()
  File "/Users/avazquez/anaconda3/envs/py34/lib/python3.4/site-packages/pgcli/main.py", line 297, in refresh_completions
    completer.extend_database_names(pgexecute.databases())
  File "/Users/avazquez/anaconda3/envs/py34/lib/python3.4/site-packages/pgcli/pgexecute.py", line 235, in databases
    cur.execute(self.databases_query)
psycopg2.ProgrammingError: column d.datcollate does not exist
LINE 4:        d.datcollate as "Collate",
               ^
```
This is with `psycopg2 v2.6`  and `pgcli v0.16`.
``` python
> import psycopg2 as pg
> pg.__version__
'2.6 (dt dec pq3 ext lo64)'

> import pgcli
> pgcli.__version__
'0.16.0'
```

I am in a python 3.4 conda environment with the latest version of Python:
```
$ python --version
Python 3.4.2 :: Anaconda 2.1.0 (x86_64)
```
 I installed `psycopg2` with `pip install pgcli` and did not get any errors. 

Any thoughts on what could be wrong?

**Comment by amjith** [comment]
What version of postgres database are you using? `pgcli` only has support for version 9.2 and above, but there is a pending issue to add support for the older versions. 

Did this issue crop up after you upgraded to the new version or is this the first time you're trying pgcli? 

I'm trying to find out if this is because of the new additions in the latest version.

**Comment by amelio-vazquez-reina** [comment]
Thank you @amjith First time using `pgcli`. Sorry, I should have mentioned that.  I am working against an older version of a PostgreSQL database, so that probably explains it:
```
> select version();
PostgreSQL 8.2.15 (Greenplum Database 4.2.6.3 build 2) on x86_64-unknown-linux-gnu, compiled by GCC gcc (GCC) 4.4.2 compiled on Oct  7 2013 15:24:14
```
What issue is the one you mention about adding support to older versions? (so that I can track it). Is it this one? https://github.com/amjith/pgcli/issues/18

Thanks.

**Comment by amjith** [comment]
Yes that's the one. 

The problem is, in order to populate the autocompletion I have to run sql queries against the database to get the list of tables and columns. That sql query is different between the various versions of the Postgres and I decided to stick with 9.2 and above since that's the one I had on my machine. :(

Now I have a few docker instances with older versions, so I'll try and get at least the basic stuff working. 

Thanks for reporting the bug. Cheers!

**Comment by amjith** [comment]
Support for postgres 8.x is now available in v1.0. Please upgrade. 

```
pip install -U pgcli
```
